.travis.yml

```diff
@@ -0,0 +1,40 @@
+language: android
+
+jdk:
+ - oraclejdk7
+ - oraclejdk8
+
+matrix:
+  allow_failures:
+    - jdk: oraclejdk7
```

Comment:
I see that with jdk7 it fails with OOM. Does it fail consistently? Because JDK7 is our releasing platform we need to make sure it builds.